lib/timestamp_utils.py

```
@@ -1,9 +1,77 @@
+    """Return the POSIX timestamp for an aware datetime."""
+    return value.timestamp()
+
+
+def utc_now() -> datetime:
+    """Return the current time as an aware UTC datetime."""
+    return datetime.now(timezone.utc)
+
+
+def format_run_timestamp(value: datetime) -> str:
+    """Format a datetime as a pipeline run-directory timestamp."""
+    return value.astimezone(timezone.utc).strftime(RUN_TIMESTAMP_FORMAT)
```

Ruff (0.16.3)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)